### DISCUSSION TITLE
When can encryption web_server_ssl_key  be supported

### DISCUSSION THREAD

**Discussion by diandian-monster** [apache/airflow#17353] [Ideas]
Encryption is not supported at present. There is a security problem

**Comment by XD-DENG** [comment]
Running the web server behind something like Nginx sounds like a proper plan IMO.

**Reply by potiuk** [reply]
What @XD-DENG  wrote is definitely recommended, however for small installations - web_server_ssl_key I believe is supported, at least for some users that I discussed with recently. @diandian-monster - can you please describe in more detiail what are you referring to? What did you try and what did not work (and give please as much information as possible if you want somoene helps you with it :).
